easybuild/easyblocks/a/atlas.py

```diff
@@ -142,9 +142,10 @@ def configure_step(self):
         # call configure in parent dir
         cmd = "%s %s/configure --prefix=%s %s" % (self.cfg['preconfigopts'], self.cfg['start_dir'],
                                                   self.installdir, self.cfg['configopts'])
-        (out, exitcode) = run_cmd(cmd, log_all=False, log_ok=False, simple=False)
+        res = run_shell_cmd(cmd, fail_on_error=False)
 
-        if exitcode != 0:
+        if res.code != 0:
```

Comment:
```suggestion
        if res.exit_code != 0:
```

Reply:
https://github.com/easybuilders/easybuild-docs/pull/240 for updating the documentation to avoid someone else making the same mistake I made here.